### DISCUSSION TITLE
Specifying non-default priors

### DISCUSSION THREAD

**Discussion by mthomas-ketchbrook** [pymc-labs/pymc-marketing#622] [Q&A] [answered]
Hi,

Similar to many others I'm sure, our team (myself & @patrick-immensityforartists) is fitting a *DelayedSaturatedMMM* model while following the [pymc-marketing documentation](https://www.pymc-marketing.io/en/stable/notebooks/mmm/mmm_example.html#model-fitting).

We are struggling with the interpretation around specifying priors ourselves (as discussed briefly in [this section](https://www.pymc-marketing.io/en/stable/notebooks/mmm/mmm_example.html#id1)), due to both the min-max scaling that takes place as well as the adstock/saturation transformations. 

If we were looking for our prior to communicate that we expect on average $5 in sales for every $1 spent on channel A, what transformations (if any) would need to take place in the 'mu' parameter for our prior specification for channel A (e.g., using a lognormal distribution specification)?  Do we need to apply min-max scaling to the mu and sigma values of our priors?

I appreciate any information that can help point us in the right direction.

Thank You!

**Comment by cetagostini** [comment, marked as answer]
Hello @mthomas-ketchbrook

Take a look here: https://github.com/pymc-labs/pymc-marketing/discussions/617#discussioncomment-9244139

This should help you to understand the function and estimate the prior, let me know if you have any other questions!

Happy to help 👍
